courseEntry.go

```diff
@@ -46,6 +46,7 @@ type CourseEntryDeleter interface {
 
 type CourseEntryService interface {
 	StoreCourseEntry(entry *CourseEntry, cfu CourseFindUpdater) (err error, courseEntry *CourseEntry)
+	StoreCourseEntryFiles(files [][]byte, id string, date time.Time) (err error, paths []string)
```

Comment:
What do you think about passing a `[]io.Reader` here instead of `[][]byte`? That would make any storing / uploading etc way more flexible.